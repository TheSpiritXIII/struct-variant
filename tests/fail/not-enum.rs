use struct_variant::struct_variant;

#[struct_variant]
struct ShapeStruct {
	value: usize,
}

fn main() {}
