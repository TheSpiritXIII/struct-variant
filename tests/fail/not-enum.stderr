error: Failed to parse struct variant input: expected `enum`
 --> $DIR/not-enum.rs:4:1
  |
4 | struct ShapeStruct {
  | ^^^^^^
